internal/stolonctl/pkg/cluster/status.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright (C) 2018 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cluster
+
+import (
+	"path/filepath"
+
+	"github.com/gravitational/stolon-app/internal/stolonctl/pkg/defaults"
+	"github.com/gravitational/stolon-app/internal/stolonctl/pkg/kubernetes"
+
+	"github.com/gravitational/stolon/pkg/cluster"
+	"github.com/gravitational/stolon/pkg/store"
+	"github.com/gravitational/trace"
+	"k8s.io/api/core/v1"
+)
+
+// getPods returns list of keeper and sentinel pods
+func getPods(config *Config) ([]v1.Pod, error) {
+	client, err := kubernetes.NewClient(config.KubeConfig)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	pods, err := client.Pods(config.KeepersPodFilter, config.Namespace)
+	if err != nil {
+		return nil, trace.Wrap(err)
```

Comment:
nit: there's a convenience function in rigging for translating from kubernetes client errors: [ConvertError](https://godoc.org/github.com/gravitational/rigging#ConvertError) that is appropriate here.